### PR TITLE
Mark web_connectivity with blocking or accessible = None as failed

### LINKED ISSUE #610: backend: mark blocking=null, accessible=null web measurements as failed
<strike>Further details will follow. Very briefly, this issue is related to https://github.com/ooni/explorer/issues/764.</strike>

This issue was actually wrong. We reverted ~immediately the change and we're working on a different solution. See below for more information on why the issue is wrong.

### PR DESCRIPTION
**Pull request by FedericoCeratto**
Fixes [backend#610](https://github.com/ooni/backend/issues/610)

Example: https://explorer.ooni.org/measurement/20220524T011649Z_webconnectivity_TM_20661_n1_K12GviVnQLLRnJoU?input=https%3A%2F%2Fajax.aspnetcdn.com%2Fajax%2F4.5.2%2F1%2FMicrosoftAjax.js

### CODE REVIEW

**Review by bassosimone — APPROVED** [review]
🐳 Thank you for implementing this! This code looks good to me!

File: af/fastpath/fastpath/core.py [hunk 1]
```diff
@@ -964,8 +964,13 @@ def score_web_connectivity(msm, matches) -> dict:
...
     #    delta = abs((tk["body_proportion"] or 1.0) - 1.0)
     #    scores["blocking_general"] += delta
-
     # TODO: refactor to apply to all test types
+
+    if "accessible" in tk and tk["accessible"] is None:
+        # https://github.com/ooni/backend/issues/610
+        scores["accuracy"] = 0.0
+        return scores
+
```
Comment (bassosimone) [inline af/fastpath/fastpath/core.py]: On re-reading, I think it would be cool to move the check for "blocking" up here as well!

**Review by bassosimone — CHANGES_REQUESTED** [review]
Sorry, after re-re-reading I think it would be better to also have "blocking" missing or None at the top of the algorithm :-)

(Please, feel free to merge once this change has been applied!)

**Review by bassosimone — APPROVED** [review]

**Comment by FedericoCeratto** [comment]
0.59~pr395-152 deployed on FSN
Includes the deployment of https://github.com/ooni/pipeline/pull/390

**Comment by FedericoCeratto** [comment]
Rolled back.

**Pull request merged**